.github/workflows/tf-aws-test.yml

```diff
@@ -0,0 +1,40 @@
+name: SDK Spec Testing
+on:
+  schedule:
+    - cron: "0 8 * * *"
+  workflow_dispatch: {}
+
+jobs:
+  install:
+    name: Installation & Test
+    strategy:
+      fail-fast: true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          repository: winglang/wing
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 19
+      - name: Configure AWS Credentials Action For GitHub Actions
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v2
+      - name: Get Latest Wing Version
+        id: get-version
+        run: |
+          echo version=$(npm view winglang version) >> $GITHUB_OUTPUT
+      - name: Install Wing
+        run: |
+          npm install -g winglang@${{ steps.get-version.outputs.version }}
+      - name: Run tests
+        run: |
+          ls
+          wing test -t tf-aws examples/tests/sdk_tests/*/*.w
```

Comment:
Checking my understanding - this will run the tests sequentially, yeah?

Maybe we can consider adding some logic so that even if a test fails, we still continue running the rest of the tests in `examples/tests/sdk_tests` so it doesn't fail at the first error...

(optional)

Reply:
Sounds good, I think I'll also generate a short summary of the results :)

Reply:
<img width="374" alt="image" src="https://github.com/winglang/wing/assets/39455181/ea1b43da-fee4-4446-9b41-fc68687fbded">

<img width="935" alt="image" src="https://github.com/winglang/wing/assets/39455181/a81765c1-b844-4c99-b814-52479fa92c43">

Reply:
this change is breaking the invalid tests (since no error is thrown), I'll fix them by reading from the log file in the snapshot instead of waiting for an error :) 
will open another PR

Reply:
done: #2638